src/style.css

```
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
don't use anything except box-sizing inside `*` selector

---

Why I cannot used it? it's not allowed under the terms.

---

It's bad for performance, fix it